kafka/src/main/resources/application.properties

```diff
@@ -23,8 +23,7 @@ timer.period = 10000
 timer.delay = 10000
 
 # Kafka instance without Authentication
-# Passing ${kafka.bootstrap.servers} is a workaround for https://github.com/apache/camel-quarkus/issues/2939
-camel.component.kafka.brokers=${brokers:${kafka.bootstrap.servers}}
+#camel.component.kafka.brokers=${brokers}
```

Comment:
https://github.com/apache/camel-quarkus/issues/2939 not being an issue anymore, I hope commenting the line back and removing ${kafka.bootstrap.servers} makes up a valid suggestion for end users?

Reply:
I think it's safe. It's how the previous impl was configured.